polling_stations/apps/data_collection/management/commands/import_west_devon.py

```diff
@@ -17,4 +17,15 @@ def address_record_to_dict(self, record):
         ]:
             rec["accept_suggestion"] = False
 
-        return rec
+        if uprn == "10013755070":
+            rec["postcode"] = "EX201FQ"
+
+    def station_record_to_dict(self, record):
+
+        # Charter Hall (Polling Station No.3)
+        if record.polling_place_id == "6524":
```

Comment:
Given there are 3 stations in the same building:
Charter Hall (Polling Station No.1)
Charter Hall (Polling Station No.2)
Charter Hall (Polling Station No.3)
now you've got a sensible UPRN and postcode, can we attach that to all of them.